README.md

```diff
@@ -1 +1,2 @@
 # My readme
+Adding some text
```

Comment:
Zsolt, what the hell were you thinking? This is terrible, pls change. Not the code, you yourself.

Reply:
Cool, will do that

Reply:
Made the changes, check again pls